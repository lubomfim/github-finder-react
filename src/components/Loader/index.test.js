import React from 'react';
import { render } from '@testing-library/react';
import { Loader } from '.';

it('should render correctly', () => {
  const { asFragment } = render(<Loader />);
  expect(asFragment(<Loader />)).toMatchSnapshot();
});
