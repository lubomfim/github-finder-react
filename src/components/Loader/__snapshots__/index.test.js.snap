// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render correctly 1`] = `
<DocumentFragment>
  .c0 {
  cursor: wait;
  width: 30px;
  height: 30px;
  border: solid 3px transparent;
  border-left-color: #24292E;
  border-right-color: #24292E;
  border-top-color: #24292E;
  -webkit-animation: rotating 0.5s 0.25s linear infinite;
  animation: rotating 0.5s 0.25s linear infinite;
  border-radius: 50%;
  margin: 0 auto;
}

<div
    class="c0"
  />
</DocumentFragment>
`;
